### DISCUSSION TITLE
Open-WebUI offline installation

### DISCUSSION THREAD

**Discussion by JeremyGarman** [open-webui/open-webui#4666] [General]
So I have a secure work environment which does not have internet access,  and requires a number of security controls and framework compliance (including hardened OS's).  I have successfully deployed Open-WebUI in the lab on the standard Ubuntu platform, as well as hardened RHEL 8 and 9 (native and using the Podman/container option).  BUT... in the non-lab environments, I won't have the option of simply curl'ing or pip'ing the different required elements for deployment.  Is there anyone who has a process (however unofficial) that worked for a fully offline deployment??

**Comment by Gaosten** [comment]
Marked，Also have this problem. BTY, did you find any solutions?😥
